src/Microsoft.Identity.Web.Certificate/DefaultCredentialsLoader.cs

```diff
@@ -116,6 +117,31 @@ public async Task LoadCredentialsIfNeededAsync(CredentialDescription credentialD
             return null;
         }
 
+        /// <inheritdoc/>
+        public async Task<SigningCredentials?> LoadSigningCredentialsAsync(
+        CredentialDescription credentialDescription,
+        CredentialSourceLoaderParameters? parameters = null)
+        {
+            _ = Throws.IfNull(credentialDescription);
+
+            try
+            {
+                await LoadCredentialsIfNeededAsync(credentialDescription, parameters);
+
+                if (credentialDescription.Certificate != null)
```

Comment:
This shows creating the SigningCredentials only from the certificate. I've included the proposal to add support for loading SymmetricSecurityKey in this PR.